distributions/openhab/src/main/resources/conf/settings/readme.txt

```diff
@@ -0,0 +1,5 @@
+Your YAML configurations go here.
+All YAML files have to have the ".yaml" file extension and must follow a special syntax.
+
+Check out the openHAB documentation for more details:
+https://www.openhab.org/docs/configuration/yaml.html
```

Comment:
Is this url correct?
https://next.openhab.org/docs/configuration/yaml.html currently gives me a 404.

Reply:
The doc hasn't been created yet, waiting for this.

Reply:
Started a draft: https://github.com/openhab/openhab-docs/pull/2508